application/src/main/java/net/corda/v5/application/interop/facade/FacadeId.java

```diff
@@ -0,0 +1,59 @@
+package net.corda.v5.application.interop.facade;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public final class FacadeId {
+
+    @NotNull
+    String owner;
```

Comment:
Is owner the right name or should we come up something more descriptive. It's not obvious to me what a FacadeId owner is

Reply:
Will add missing javadocs to provide explanation